### PR TITLE
[rust] Add RsSymbolBlock

### PR DESCRIPTION
**Pull request by xyang16**
## Description ##

Brief description of what this PR is about

- If this change is a backward incompatible change, why must this change be made?
- Interesting edge cases to note here

### CODE REVIEW

File: extensions/tokenizers/src/main/java/ai/djl/engine/rust/jni/JniUtils.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+import ai.djl.engine.rust.RustLibrary;
+import ai.djl.ndarray.NDList;
+
+import java.util.List;
+
+/**
+ * A class containing utilities to interact with the Rust Engine's Java Native Interface (JNI)
+ * layer.
+ */
+@SuppressWarnings("MissingJavadocMethod")
```
Comment (frankfliu) [inline extensions/tokenizers/src/main/java/ai/djl/engine/rust/jni/JniUtils.java:26]: We don't really need this wrap. We can call RustLibrary directly.

File: extensions/tokenizers/rust/src/compute_cap.rs [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    CU_DEVICE_ATTRIBUTE_COMPUTE_CAPABILITY_MAJOR, CU_DEVICE_ATTRIBUTE_COMPUTE_CAPABILITY_MINOR,
+};
+use candle::cuda_backend::cudarc::driver::CudaDevice;
+use std::sync::Once;
+
+static INIT: Once = Once::new();
+static mut RUNTIME_COMPUTE_CAP: usize = 0;
+static mut COMPILE_COMPUTE_CAP: usize = 0;
+
+fn init_compute_caps() {
```
Comment (frankfliu) [inline extensions/tokenizers/rust/src/compute_cap.rs:11]: Need implement RsEngine.hasCapability() using this function.

File: extensions/tokenizers/rust/src/models/distilbert.rs [hunk 3]
```diff
@@ -0,0 +1,390 @@
+use crate::models::Model;
```
Comment (lanking520) [inline extensions/tokenizers/rust/src/models/distilbert.rs:1]: why modelling file is under tokenizer?

**Review by lanking520 — COMMENTED** [review]
It's a bit strange to use tokenizer module as the holder for Rust engine. Could we merge with Candle project?

File: extensions/tokenizers/src/main/java/ai/djl/engine/rust/jni/JniUtils.java (hunk 1 above)
Comment (xyang16) [reply extensions/tokenizers/src/main/java/ai/djl/engine/rust/jni/JniUtils.java]: Done

File: extensions/tokenizers/rust/src/compute_cap.rs (hunk 2 above)
Comment (xyang16) [reply extensions/tokenizers/rust/src/compute_cap.rs:11]: Added

**Review by frankfliu — APPROVED** [review]

**Pull request merged**